pre_award/db/migrations/versions/010_data_standards.py

```diff
@@ -0,0 +1,43 @@
+"""Enforcing data standards:
+- Rename eoi_decision_schema to remove space in column name
+- Change owner_organisation_logo_uri to String type
+
+Revision ID: 010_
+Revises: 009_relax_flag_json_constraints
+Create Date: 2025-01-29 09:50:21.026372
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "010_"
```

Comment:
might be nice for this to have a sensible revision id, although I appreciate it's _close_ to a noop